python/requirements/ml/requirements_rllib.txt

```diff
@@ -15,7 +15,7 @@ pettingzoo==1.11.1
 pymunk==6.0.0
 supersuit==2.6.6
 # For testing in MuJoCo-like envs (in PyBullet).
-pybullet==3.1.7
+pybullet==3.2.0
```

Comment:
@gjoliver @richardliaw : I'm a little nervous about this change (since 1.9 has run into a few dependency related issues). What's your take on the risk?

Is there an option to just verify `learning_tests` can pass with this, but without actually cherry picking it to 1.9?

Reply:
we can definitely cherry pick it to a test branch based on r1.9 branch and run learning test from there.
what do you think?

Reply:
+1 on test branch

But my question still stands: even after the verification on the test branch, do we need to cherry pick this PR to 1.9 release? If the only purpose is to fix `learning_tests`, is it OK to leave it out to reduce risk (e.g. what if the new pybullet introduces other risks)? Thanks

Reply:
@zhe-thoughts I agree- this doesn't need to be picked into 1.9 release. It's not fixing anything with rllib itself.